app/javascript/flavours/glitch/features/compose/components/compose_form.jsx

```diff
@@ -322,10 +320,10 @@ class ComposeForm extends ImmutablePureComponent {
             onKeyDown={this.handleKeyDown}
             disabled={!spoiler}
             ref={this.handleRefSpoilerText}
-            suggestions={this.props.suggestions}
+            suggestions={suggestions}
             onSuggestionsFetchRequested={onFetchSuggestions}
             onSuggestionsClearRequested={onClearSuggestions}
-            onSuggestionSelected={this.onSpoilerSuggestionSelected}
+            onSuggestionSelected={onSpoilerSuggestionSelected}
```

Comment:
Am I missing something or does this code mean autosuggestions should also work in the spoiler field?
Because they don't.
But it also didn't work before.

Reply:
It should suggest custom emojis.

Reply:
Oooooh, I forgot that is a thing. Makes sense.
Also same thing about TypeError.